src/c75_menuCTRL.hpp

```diff
@@ -130,10 +235,28 @@ bool processControlKeys()
                 {
                     inSerialControl = !inSerialControl;
                 }
-                else if ((key == btnDOWN) || (key == btnUP))
+                else if ((key == btnDOWN))
                 {
                     ctrlState = HIGHLIGHT_MANUAL;
                 }
+                else if (key == btnUP)
+                {
+        #if USE_HALL_SENSOR_RA_AUTOHOME == 1
+                    ctrlState = HIGHLIGHT_RA_AUTO_HOME;
+        #else
+                    ctrlState = HIGHLIGHT_MANUAL;
+        #endif
+                }
+                //DEC-
+                else if (key == btnUP)
```

Comment:
Again, will never run

Reply:
That one might actually be broken, I did say it in the Discord channel...